recipe/meta.yaml

```diff
@@ -256,27 +248,22 @@ outputs:
         - wheel
       run:
         - python
-        - numpy <1.29,>=1.21
-        - scipy <1.13,>=1.5.4
-        - pandas <2.2.0,>=2.0.0
-        - scikit-learn <1.5,>=1.3.0
-        - networkx <4,>=3.0
-        - {{ pin_subpackage(name ~ '.core', exact=True) }}
-        - {{ pin_subpackage(name ~ '.features', exact=True) }}
-        # These are for the extra group "all".
-        - xgboost <2.1,>=1.6
-        - {{ pin_subpackage(name ~ '.core', exact=True) }}
-        - pytorch <2.1,>=2.0
-        - fastai <2.8,>=2.3.1
-        - lightgbm <4.2,>=3.3
-        - catboost <1.3,>=1.1
+        - {{ pin_subpackage('autogluon.core', exact=True) }}
+        - {{ pin_subpackage('autogluon.features', exact=True) }}
+        # Extra group "all". autogluon requires autogluon.tabular[all].
+        - lightgbm >=3.3,<4.4
+        - catboost >=1.1,<1.3
```

Comment:
this should be
```
- catboost >=1.1,<1.2  # [osx and py<311]
- catboost >=1.1,<1.3  # [not (osx and py<311)]
```
according to https://github.com/autogluon/autogluon/blob/v1.1.1/tabular/setup.py

Reply:
`grayskull` let me down... again 😞

Reply:
@boldorider4 We don't have `catboost >=1.1,<1.2`. I did notice that [this provision](https://github.com/autogluon/autogluon/blob/v1.0.0/tabular/setup.py#L42) is present in `autogluon 1.0.0` as well, which we built [without it](https://github.com/AnacondaRecipes/autogluon-feedstock/blob/master/recipe/meta.yaml#L272) with no apparent issues.